src/test/java/software/amazon/cloudformation/resource/ResourceTypeSchemaTest.java

```diff
@@ -101,6 +101,16 @@ public void getWriteOnlyProperties() {
         assertThat(result).containsExactly("/properties/propertyC", "/properties/propertyE/nestedProperty");
     }
 
+    @Test
+    public void getReplacementStrategy() {
```

Comment:
can we have a test that tests the opposite [delete, create] as well? just would be good for sanity to know that this deserializes correctly in both cases.

Reply:
👌will add

Reply:
> can we have a test that tests the opposite [delete, create] as well? just would be good for sanity to know that this deserializes correctly in both cases.
👌will add